src/routes/comments.js

```diff
@@ -60,4 +62,34 @@ router.get('/', async (req, res) => {
   }
 })
 
+// delete a comment
+router.delete('/:comment_id', auth, async (req, res) => {
+  try {
+    // confirm comment to delete exists in db
+    const comment = await Comment.findOne({
+      where: { id: req.params.comment_id }
+    })
+
+    if (!comment) {
+      return res.status(404).json({ msg: 'Comment not found' })
+    }
+
+    // confirm comment to delete is owned by user who is deleting
+    if (comment.dataValues.user_id !== req.user.id) {
+      return res.status(401).json({ msg: 'Cannot delete other users comments' })
+    }
+
+    // delete the comment from the db
+    await comment.destroy({ force: true })
+
+    res.status(200).json({ msg: 'Comment deleted' })
+  } catch (err) {
+    if (err.message.match(/^(invalid input syntax for integer)/)) {
+      // this message will trigger iff non integer is put in req param
```

Comment:
if*

Reply:
@Formless iff = if and only if. don't you remember your discreet math?

Reply:
discrete* lol